source/code/go/src/plugins/telemetry.go

```diff
@@ -138,3 +144,10 @@ func SendEvent(eventName string, dimensions map[string]string) {
 
 	TelemetryClient.Track(event)
 }
+
+// SendException  send an event to the configured app insights instance
+func SendException(err interface{}) {
+	if TelemetryClient != nil {
+		TelemetryClient.TrackException(err)
```

Comment:
We need all dimensions like we have for events & metrics.

Reply:
will get it automatically, since the common dimensions are on the telemetry client object.